NetTopologySuite.IO.SqlServer.Test/NetTopologySuite.IO.SqlServer.Test.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net472;netcoreapp2.1</TargetFrameworks>
+    <IsPackable>false</IsPackable>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\nts.snk</AssemblyOriginatorKeyFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.8.0" />
+    <PackageReference Include="NetTopologySuite.Core" Version="1.15.1" />
+    <PackageReference Include="xunit" Version="2.4.0" />
```

Comment:
Should I port these to NUnit?

Reply:
I don't think it's necessary, especially not right away.